src/base/_foundation.lua

```diff
@@ -34,6 +34,7 @@
 	premake.CSHARP      = "C#"
 	premake.GCC         = "gcc"
 	premake.HAIKU       = "haiku"
+    premake.ANDROID     = "android"
```

Comment:
Drive by review, but there's some minor indentation issues here

Reply:
oops, will fix it now